### DISCUSSION TITLE
Locales get duplicated after deleting a locale

### DISCUSSION THREAD

**Discussion by emilsbee** [swellstores/community#445] [Help] [answered]
I am currently facing the following situation
![image](https://github.com/swellstores/community/assets/31566289/3b87ceec-89bd-4a92-ab5b-6f2282a644da)
When I attempt to delete the Azerbaijan language then another Catalan language will be created in its place. I just cannot delete these locales. Anybody else faced something like this?

**Comment by logeshswell** [comment]
Can you help me with your store URL?

**Reply by logeshswell** [reply]
It looks like your store data is corrupted with duplicate values and hence I couldn't reproduce this issue in my store.
But, I can fix it at our end if you provide your store URL.

**Reply by emilsbee** [reply]
Sure, it's https://emils-store-2.swell.store/

**Reply by logeshswell** [reply, marked as answer]
Thanks, I can reproduce the issue in test env of your store. 
I can notice that the locale data is corrupted when we attempt to delete locales in test env.
We'll take a deeper look at this issue and get back to you if I get some updates.

**Reply by logeshswell** [reply]
For the time-being, I removed all locales in test env of your store. Please add the required locales in your Swell store.
Don't try to delete any locale until the fix has been made.

**Reply by emilsbee** [reply]
Thanks for removing the duplicate locales and looking into the issue. I won't be deleting any more until the fix.

**Reply by logeshswell** [reply]
The bug fix is rolled out now. Can you check and confirm if it's working fine for you?
